examples/observability/simple_calculator_observability/tests/test_simple_calc_observability.py

```
@@ -0,0 +1,107 @@
+@pytest.fixture(name="question", scope="module")
+def question_fixture() -> str:
+    return "What is 2 * 4?"
+
+
+@pytest.fixture(name="expected_answer", scope="module")
+def expected_answer_fixture() -> str:
+    return "8"
+
+
+@pytest.fixture(name="weave_project_name", scope="module")
+async def fixture_weave_project_name(weave: types.ModuleType, wandb_api_key) -> AsyncGenerator[str]:
```

Ruff (0.14.0)

Unused function argument: `wandb_api_key`

(ARG001)